Tests/Python/PandasConverterTests.cs

```diff
@@ -223,6 +223,47 @@ public void HandlesNullableValues()
             }
         }
 
+        /// <summary>
+        /// Specific issues for symbol LOW, reference GH issue #4886
+        /// </summary>
+        [Test]
+        public void HandlesOddTickers()
+        {
+            var converter = new PandasConverter();
+            var symbol = Symbols.LOW;
+
+            var rawBars = Enumerable
+                .Range(0, 10)
+                .Select(i => new TradeBar(DateTime.UtcNow.AddMinutes(i), symbol, i + 101m, i + 102m, i + 100m, i + 101m, 0m))
+                .ToArray();
+
+            // GetDataFrame with argument of type IEnumerable<TradeBar>
+            var history = GetHistory(symbol, Resolution.Minute, rawBars);
+            dynamic dataFrame = converter.GetDataFrame(history);
+
+            // Add LOW to our symbol cache
+            SymbolCache.Set("LOW", Symbols.LOW);
+
+            using (Py.GIL())
+            {
+
+                dynamic test = PythonEngine.ModuleFromString("testModule",
+    $@"
+def Test(dataFrame):
+    data = dataFrame.loc['LOW']
+    if data.empty:
+        raise Exception('LOW history data is empty')
+    if data.__len__() != 10:
+        raise Exception('Expected 10 data points')
+    lowData = data.low
+    if data.empty is 0:
```

Comment:
`if data.empty is 0` should check `lowData` instead?

Reply:
My bad, thanks boss!!!!